--- audacity-src-1.2.3/src/MeterToolBar.cpp.orig	2004-11-18 07:17:30.000000000 +0100
+++ audacity-src-1.2.3/src/MeterToolBar.cpp	2005-05-03 15:11:28.885869256 +0200
@@ -16,6 +16,7 @@
 #include <wx/object.h> // tooltip.h needs this but doesn't include it.
 #include <wx/window.h> // tooltip.h needs this but doesn't include it.
 #include <wx/tooltip.h>
+#include <wx/dcclient.h>
 
 #include "MeterToolBar.h"
 
