--- audacity-minsrc-2.4.2/lib-src/portaudio-v19/include/pa_mac_core.h.old	2018-08-28 10:48:03.000000000 +0000
+++ audacity-minsrc-2.4.2/lib-src/portaudio-v19/include/pa_mac_core.h	2021-01-18 00:32:30.022599655 +0000
@@ -45,8 +45,8 @@
 
 #include "portaudio.h"
 
-#include <AudioUnit/AudioUnit.h>
-#include <AudioToolbox/AudioToolbox.h>
+//#include <AudioUnit/AudioUnit.h>
+//#include <AudioToolbox/AudioToolbox.h>
 
 #ifdef __cplusplus
 extern "C" {
