diff -aurN audacity-src-1.2.3.orig/lib-src/allegro/Makefile.in audacity-src-1.2.3/lib-src/allegro/Makefile.in
--- audacity-src-1.2.3.orig/lib-src/allegro/Makefile.in	2004-11-18 07:17:30.000000000 +0100
+++ audacity-src-1.2.3/lib-src/allegro/Makefile.in	2004-12-05 19:50:14.926174384 +0100
@@ -3,7 +3,7 @@
 CC = @CC@
 CCC = @CXX@
 
-CFLAGS = -g -O
+CFLAGS = @CFLAGS@
 OBJ = \
 	allegro.cpp.o \
 	allegrord.cpp.o \
diff -aurN audacity-src-1.2.3.orig/lib-src/libnyquist/Makefile.in audacity-src-1.2.3/lib-src/libnyquist/Makefile.in
--- audacity-src-1.2.3.orig/lib-src/libnyquist/Makefile.in	2004-11-18 07:17:32.000000000 +0100
+++ audacity-src-1.2.3/lib-src/libnyquist/Makefile.in	2004-12-05 19:51:30.418697776 +0100
@@ -6,7 +6,7 @@
 INCL = -Inyqsrc -Itran -Ixlisp -Icmt -Isys -Isnd -Ifft
 
 CC = @CC@
-override CFLAGS += -g -DCMTSTUFF -O2 -DEXT $(INCL)
+override CFLAGS += $(OPTFLAGS) -DCMTSTUFF -DEXT $(INCL)
 LN = $(CC)
 LFLAGS = -lm
 
diff -aurN audacity-src-1.2.3.orig/lib-src/portaudio/pa_unix_oss/Makefile audacity-src-1.2.3/lib-src/portaudio/pa_unix_oss/Makefile
--- audacity-src-1.2.3.orig/lib-src/portaudio/pa_unix_oss/Makefile	2004-11-18 07:17:34.000000000 +0100
+++ audacity-src-1.2.3/lib-src/portaudio/pa_unix_oss/Makefile	2004-12-05 19:52:26.867116312 +0100
@@ -2,7 +2,7 @@
 
 CC = gcc
 
-override CFLAGS += -g -O -I../pa_common
+override CFLAGS += $(OPTFLAGS) -I../pa_common
 PASRC = ../pa_common/pa_lib.c pa_unix_oss.c pa_unix.c
 PAINC = ../pa_common/portaudio.h
 PAOBJ = ../pa_common/pa_lib.o pa_unix_oss.o pa_unix.o
diff -aurN audacity-src-1.2.3.orig/lib-src/portaudio-v19/Makefile.linux audacity-src-1.2.3/lib-src/portaudio-v19/Makefile.linux
--- audacity-src-1.2.3.orig/lib-src/portaudio-v19/Makefile.linux	2004-11-18 07:17:34.000000000 +0100
+++ audacity-src-1.2.3/lib-src/portaudio-v19/Makefile.linux	2004-12-05 19:53:38.559217456 +0100
@@ -17,7 +17,7 @@
 LIBS =  -lm -lpthread
 
 CDEFINES = -I../pa_common
-CFLAGS = -g
+CFLAGS = $(OPTFLAGS)
 LIBINST = /usr/local/lib
 
 TESTS:= $(wildcard pa_tests/pa*.c pa_tests/debug*.c)
diff -aurN audacity-src-1.2.3.orig/lib-src/soundtouch/configure.ac audacity-src-1.2.3/lib-src/soundtouch/configure.ac
--- audacity-src-1.2.3.orig/lib-src/soundtouch/configure.ac	2004-11-18 07:17:35.000000000 +0100
+++ audacity-src-1.2.3/lib-src/soundtouch/configure.ac	2004-12-05 20:01:48.171785032 +0100
@@ -35,7 +35,7 @@
 dnl ############################################################################
 dnl # Checks for programs                                                      #
 dnl ############################################################################
-CXXFLAGS= dnl AC_PROG_CXX automatically sets CXXFLAGS to -g -O2 if g++ is the C++ compiler and CXXFLAGS hasn't been set, so by setting CXXFLAGS, we can avoid this.
+CXXFLAGS=$(OPTFLAGS)
 AC_PROG_CXX
 #AC_PROG_AWK
 AC_PROG_CC
