diff -aurN audacity-src-1.2.0-pre2.orig/src/Help.cpp audacity-src-1.2.0-pre2/src/Help.cpp
--- audacity-src-1.2.0-pre2.orig/src/Help.cpp	2003-09-29 08:51:27.000000000 +0200
+++ audacity-src-1.2.0-pre2/src/Help.cpp	2003-10-16 13:54:11.380095056 +0200
@@ -48,7 +48,7 @@
       if (helpFiles.GetCount() > 0)
          defaultLoc = helpFiles[0];
       else
-         defaultLoc = INSTALL_PREFIX "/share/doc/audacity/audacity-1.2-help.htb";
+         defaultLoc = INSTALL_PREFIX "/share/audacity/audacity-1.2-help.htb";
 
       wxString helpFilePath =
           gPrefs->Read("/Help/HelpFilePath1.2", defaultLoc);
