--- audacity-src-1.3.12-beta/lib-src/portaudio-v19/include/pa_mac_core.h~	2010-03-30 13:28:56.000000000 +0200
+++ audacity-src-1.3.12-beta/lib-src/portaudio-v19/include/pa_mac_core.h	2010-05-16 12:51:23.740249675 +0200
@@ -43,7 +43,7 @@
  *  @brief CoreAudio-specific PortAudio API extension header file.
  */
 
-#include <AudioUnit/AudioUnit.h>
+//#include <AudioUnit/AudioUnit.h>
 //#include <AudioToolbox/AudioToolbox.h>
 
 #ifdef __cplusplus
