--- audacity-src-1.0.0/configure.in.orig	Wed Jun  5 09:51:19 2002
+++ audacity-src-1.0.0/configure.in	Wed Jul 24 21:05:39 2002
@@ -56,7 +56,7 @@
 	use_vorbis="no")
 AC_ARG_WITH(id3,
 	[AC_HELP_STRING([--with-id3],
-		[compile with id3lib (the source is included here) [default=no]])],
+		[compile with id3lib (libraries must be installed) [default=no]])],
 	use_id3lib=$withval,
 	use_id3lib="no")
 AC_ARG_WITH(help,
@@ -161,9 +161,7 @@
 fi
 
 if [[ $use_id3lib = "yes" ]] ; then
-	EXTRAOBJS="$EXTRAOBJS id3lib/src/.libs/libid3.a"
-	MAKEALL="$MAKEALL id3lib/src/.libs/libid3.a"
-	CFLAGS="$CFLAGS -Iid3lib/include"
+	LIBS="$LIBS -lid3"
 		
 	AC_DEFINE(USE_ID3LIB, 1,
       [Define if id3lib is present])
