--- audacity-src-1.1.1-3/src/Help.cpp.orig	Mon Nov 11 07:58:21 2002
+++ audacity-src-1.1.1-3/src/Help.cpp	Sat Jan 25 01:08:40 2003
@@ -81,7 +81,7 @@
       if (helpFiles.GetCount() > 0)
          defaultLoc = helpFiles[0];
       else
-         defaultLoc = INSTALL_PREFIX "/share/doc/audacity/audacity-help.htb";
+         defaultLoc = INSTALL_PREFIX "/share/audacity/audacity-help.htb";
 
       wxString helpFilePath =
           gPrefs->Read("/Help/HelpFilePath", defaultLoc);
