--- audacity-minsrc-2.4.2/src/audacity.desktop.in.old	2019-10-21 20:29:41.000000000 +0000
+++ audacity-minsrc-2.4.2/src/audacity.desktop.in	2021-01-18 11:58:08.159386601 +0000
@@ -12,6 +12,7 @@
 GenericName[ko]=오디오 편집기
 GenericName[lt]=Garsų rengyklė
 GenericName[nl]=Geluidseditor
+GenericName[pl]=Edytor plików dźwiękowych
 GenericName[pt_BR]=Editor de áudio
 GenericName[pt_PT]=Editor de áudio
 GenericName[ru]=Редактор звуковых файлов
@@ -32,6 +33,7 @@
 Comment[kr]=오디오 파일 녹음과 편집
 Comment[lt]=Įrašyti ir montuoti garso failus
 Comment[nl]=Audiobestanden opnemen en bewerken
+Comment[pl]=Edytor plików dźwiękowych
 Comment[pt_BR]=Gravar e editar arquivos de áudio
 Comment[pt_PT]=Gravar e editar ficheiros de áudio
 Comment[ru]=Запись и редактирование звуковых файлов
