--- audacity-src-1.1.3/lib-src/allegro/Makefile.in.wiget	Thu Mar 20 05:38:08 2003
+++ audacity-src-1.1.3/lib-src/allegro/Makefile.in	Fri Mar 28 01:56:59 2003
@@ -3,7 +3,7 @@
 CC = @CC@
 CCC = @CXX@
 
-CFLAGS = -g -O
+CFLAGS = @CFLAGS@
 OBJ = \
 	allegro.cpp.o \
 	allegrord.cpp.o \
--- audacity-src-1.1.3/lib-src/libnyquist/Makefile.wiget	Thu Mar 20 05:38:11 2003
+++ audacity-src-1.1.3/lib-src/libnyquist/Makefile	Fri Mar 28 01:58:03 2003
@@ -6,7 +6,7 @@
 INCL = -Inyqsrc -Itran -Ixlisp -Icmt -Isys -Isnd -Ifft
 
 CC = cc
-override CFLAGS += -g -DCMTSTUFF -O2 -DEXT $(INCL)
+override CFLAGS += $(OPTFLAGS) -DCMTSTUFF -DEXT $(INCL)
 LN = $(CC)
 LFLAGS = -lm
 
--- audacity-src-1.1.3/lib-src/portaudio/pa_unix_oss/Makefile.wiget	Thu Mar 20 05:38:17 2003
+++ audacity-src-1.1.3/lib-src/portaudio/pa_unix_oss/Makefile	Fri Mar 28 01:58:45 2003
@@ -2,7 +2,7 @@
 
 CC = gcc
 
-override CFLAGS += -g -O -ansi -I../pa_common
+override CFLAGS += $(OPTFLAGS) -ansi -I../pa_common
 PASRC = ../pa_common/pa_lib.c pa_unix_oss.c pa_unix.c
 PAINC = ../pa_common/portaudio.h
 PAOBJ = ../pa_common/pa_lib.o pa_unix_oss.o pa_unix.o
