diff -urN audacity-src-0.98.orig/ExportMP3.cpp audacity-src-0.98/ExportMP3.cpp
--- audacity-src-0.98.orig/ExportMP3.cpp	Tue Jan 22 07:14:15 2002
+++ audacity-src-0.98/ExportMP3.cpp	Sat Feb  2 15:59:19 2002
@@ -212,12 +212,12 @@
          
          wxString GetLibraryName()
          {
-            return "libmp3lame.so";
+            return "libmp3lame.so.0.0.0";
          }
          
          wxString GetLibraryTypeString()
          {
-            return wxString("Shared Object files (*.so)|*.so");
+            return wxString("Shared Object files (*.so*)|*.so*");
          }
          
          wxString GetLibraryMessage()
