--- audacity-src-1.1.1-3/lib-src/allegro/Makefile.in.orig	Mon Nov 11 07:58:21 2002
+++ audacity-src-1.1.1-3/lib-src/allegro/Makefile.in	Sat Jan 25 01:31:51 2003
@@ -3,7 +3,7 @@
 CC = @CC@
 CCC = @CXX@
 
-CFLAGS = -g -O
+CFLAGS = @CFLAGS@
 OBJ = \
 	allegro.cpp.o \
 	allegrord.cpp.o \
--- audacity-src-1.1.1-3/lib-src/libnyquist/Makefile.orig	Mon Nov 11 07:58:21 2002
+++ audacity-src-1.1.1-3/lib-src/libnyquist/Makefile	Sat Jan 25 01:33:47 2003
@@ -6,7 +6,7 @@
 INCL = -Inyqsrc -Itran -Ixlisp -Icmt -Isys -Isnd -Ifft
 
 CC = cc
-CFLAGS = -g -DCMTSTUFF -O2 -DEXT $(INCL)
+CFLAGS = $(OPTFLAGS) -DCMTSTUFF -DEXT $(INCL)
 LN = $(CC)
 LFLAGS = -lm
 
--- audacity-src-1.1.1-3/lib-src/portaudio/pa_unix_oss/Makefile.orig	Mon Nov 11 07:58:21 2002
+++ audacity-src-1.1.1-3/lib-src/portaudio/pa_unix_oss/Makefile	Sat Jan 25 01:38:35 2003
@@ -2,7 +2,7 @@
 
 CC = gcc
 
-CFLAGS = -g -O -ansi -I../pa_common
+CFLAGS = $(OPTFLAGS) -ansi -I../pa_common
 PASRC = ../pa_common/pa_lib.c pa_unix_oss.c
 PAINC = ../pa_common/portaudio.h
 PAOBJ = ../pa_common/pa_lib.o pa_unix_oss.o
