diff -aurN audacity-src-1.2.0-pre2.orig/lib-src/allegro/Makefile.in audacity-src-1.2.0-pre2/lib-src/allegro/Makefile.in
--- audacity-src-1.2.0-pre2.orig/lib-src/allegro/Makefile.in	2003-09-29 08:51:27.000000000 +0200
+++ audacity-src-1.2.0-pre2/lib-src/allegro/Makefile.in	2003-10-16 13:56:45.234705600 +0200
@@ -3,7 +3,7 @@
 CC = @CC@
 CCC = @CXX@
 
-CFLAGS = -g -O
+CFLAGS = @CFLAGS@
 OBJ = \
 	allegro.cpp.o \
 	allegrord.cpp.o \
diff -aurN audacity-src-1.2.0-pre2.orig/lib-src/libnyquist/Makefile.in audacity-src-1.2.0-pre2/lib-src/libnyquist/Makefile.in
--- audacity-src-1.2.0-pre2.orig/lib-src/libnyquist/Makefile.in	2003-09-29 08:51:31.000000000 +0200
+++ audacity-src-1.2.0-pre2/lib-src/libnyquist/Makefile.in	2003-10-16 13:57:45.269578912 +0200
@@ -6,7 +6,7 @@
 INCL = -Inyqsrc -Itran -Ixlisp -Icmt -Isys -Isnd -Ifft
 
 CC = @CC@
-override CFLAGS += -g -DCMTSTUFF -O2 -DEXT $(INCL)
+override CFLAGS += $(OPTFLAGS) -DCMTSTUFF -DEXT $(INCL)
 LN = $(CC)
 LFLAGS = -lm
 
diff -aurN audacity-src-1.2.0-pre2.orig/lib-src/portaudio/pa_unix_oss/Makefile audacity-src-1.2.0-pre2/lib-src/portaudio/pa_unix_oss/Makefile
--- audacity-src-1.2.0-pre2.orig/lib-src/portaudio/pa_unix_oss/Makefile	2003-09-29 08:51:36.000000000 +0200
+++ audacity-src-1.2.0-pre2/lib-src/portaudio/pa_unix_oss/Makefile	2003-10-16 13:58:31.298581440 +0200
@@ -2,7 +2,7 @@
 
 CC = gcc
 
-override CFLAGS += -g -O -I../pa_common
+override CFLAGS += $(OPTFLAGS) -I../pa_common
 PASRC = ../pa_common/pa_lib.c pa_unix_oss.c pa_unix.c
 PAINC = ../pa_common/portaudio.h
 PAOBJ = ../pa_common/pa_lib.o pa_unix_oss.o pa_unix.o
diff -aurN audacity-src-1.2.0-pre2.orig/lib-src/portaudio-v19/Makefile.linux audacity-src-1.2.0-pre2/lib-src/portaudio-v19/Makefile.linux
--- audacity-src-1.2.0-pre2.orig/lib-src/portaudio-v19/Makefile.linux	2003-09-29 08:51:37.000000000 +0200
+++ audacity-src-1.2.0-pre2/lib-src/portaudio-v19/Makefile.linux	2003-10-16 13:59:07.003153520 +0200
@@ -17,7 +17,7 @@
 LIBS =  -lm -lpthread
 
 CDEFINES = -I../pa_common
-CFLAGS = -g
+CFLAGS = $(OPTFLAGS)
 LIBINST = /usr/local/lib
 
 TESTS:= $(wildcard pa_tests/pa*.c pa_tests/debug*.c)
diff -aurN audacity-src-1.2.0-pre2.orig/lib-src/soundtouch/configure.in audacity-src-1.2.0-pre2/lib-src/soundtouch/configure.in
--- audacity-src-1.2.0-pre2.orig/lib-src/soundtouch/configure.in	2003-09-29 08:51:38.000000000 +0200
+++ audacity-src-1.2.0-pre2/lib-src/soundtouch/configure.in	2003-10-16 13:59:31.945361728 +0200
@@ -24,6 +24,6 @@
     AC_MSG_ERROR("Could not find ar - needed to create a library");
 fi
 
-CXXFLAGS="-O2"
+CXXFLAGS=$(OPTFLAGS)
 
 AC_OUTPUT([Makefile src/Makefile])
