--- audacity-src-1.1.3/src/xml/XMLFileReader.h.wiget	Thu Mar 20 05:38:08 2003
+++ audacity-src-1.1.3/src/xml/XMLFileReader.h	Fri Mar 28 01:37:22 2003
@@ -8,7 +8,7 @@
 
 **********************************************************************/
 
-#include "xmlparse/xmlparse.h"
+#include <expat.h>
 
 #include "XMLTagHandler.h"
 
--- audacity-src-1.1.3/configure.in.wiget	Thu Mar 20 05:38:07 2003
+++ audacity-src-1.1.3/configure.in	Fri Mar 28 01:37:56 2003
@@ -87,9 +87,7 @@
 dnl Required sublibraries
 dnl
 
-BUILDLIBS="$BUILDLIBS expat/expat.a"
-EXTRAOBJS="$EXTRAOBJS ../lib-src/expat/expat.a"
-CXXFLAGS="$CXXFLAGS -I../lib-src/expat"
+LIBS="$LIBS -lexpat"
 
 dnl TODO enable/disable allegro
 BUILDLIBS="$BUILDLIBS allegro/allegro.a"
@@ -486,7 +484,7 @@
 
 AC_CONFIG_HEADER(src/configunix.h:src/configtemplate.h)
 AC_CONFIG_FILES([Makefile src/Makefile lib-src/Makefile \
-					  lib-src/allegro/Makefile lib-src/expat/Makefile
+					  lib-src/allegro/Makefile 
                                           locale/Makefile tests/Makefile])
 
 AC_CONFIG_SUBDIRS([$SUBDIRS])
--- audacity-src-1.1.3/src/commands/CommandsReader.h.wigett	Fri Mar 28 16:34:56 2003
+++ audacity-src-1.1.3/src/commands/CommandsReader.h	Fri Mar 28 16:35:10 2003
@@ -13,7 +13,7 @@
 #ifndef __AUDACITY_COMMANDS_READER__
 #define __AUDACITY_COMMANDS_READER__
 
-#include "xmlparse/xmlparse.h"
+#include <expat.h>
 #include "CommandsMenu.h"
 
 struct CommandsTranslationItem
