--- audacity-src-1.1.1-3/src/Prefs.cpp.orig	Mon Nov 11 07:58:21 2002
+++ audacity-src-1.1.1-3/src/Prefs.cpp	Sat Jan 25 02:43:25 2003
@@ -116,7 +116,9 @@
    if(prefsversion.CmpNoCase((wxString)AUDACITY_PREFS_VERSION_STRING))
    {
       // BG: Reset the prefs by removing them
+      if(gPrefs->HasGroup("/Keyboard"))
       gPrefs->DeleteGroup("/Keyboard");
+      if(gPrefs->HasGroup("/Locale"))
       gPrefs->DeleteGroup("/Locale");
       gPrefs->Write("/PrefsVersion", (wxString)AUDACITY_PREFS_VERSION_STRING);
    }
