--- audacity-src-1.2.4b.orig/lib-src/soundtouch/include/SoundTouch.h	2005-11-27 03:33:39.000000000 +0100
+++ audacity-src-1.2.4b/lib-src/soundtouch/include/SoundTouch.h	2006-08-30 18:29:27.075245750 +0200
@@ -146,7 +146,7 @@
     static const char *getVersionString();
 
     /// Get SoundTouch library version Id
-    static uint SoundTouch::getVersionId();
+    static uint getVersionId();
 
     /// Sets new rate control value. Normal rate = 1.0, smaller values
     /// represent slower rate, larger faster rates.
